### PR TITLE
CAMEL-10334 Use of whitespaces in remotePath

### PR DESCRIPTION
**Pull request by undso**
https://issues.apache.org/jira/browse/CAMEL-10334

### CODE REVIEW

File: components/camel-dropbox/src/main/java/org/apache/camel/component/dropbox/DropboxComponent.java [hunk 1]
```diff
@@ -50,7 +50,7 @@ protected Endpoint createEndpoint(String uri, String remaining, Map<String, Obje
         // set options from component
         configuration.setAccessToken((String)parameters.get("accessToken"));
         configuration.setLocalPath((String)parameters.get("localPath"));
-        configuration.setRemotePath((String)parameters.get("remotePath"));
+        configuration.setRemotePath(((String)parameters.get("remotePath")).replaceAll("\\s","+"));
```
Comment (davsclaus) [inline components/camel-dropbox/src/main/java/org/apache/camel/component/dropbox/DropboxComponent.java]: Can you check for null, if removePath is null.

Comment (undso) [inline components/camel-dropbox/src/main/java/org/apache/camel/component/dropbox/DropboxComponent.java]: Good point. Thank you for the reminder. Check is added.

**Comment by davsclaus** [comment]
Looks good @oscerd can you merge?

**Comment by oscerd** [comment]
Sure

**Comment by oscerd** [comment]
@undos thanks! The PR has been merged. Can you close this one?

**Pull request closed without merging**